src/v3/test/integration/__snapshots__/authenticator-enroll-data-phone.test.tsx.snap

```diff
@@ -1500,13 +1498,27 @@ exports[`authenticator-enroll-data-phone should render form 1`] = `
                             d="M7 10l5 5 5-5z"
                           />
                         </svg>
+                        <fieldset
+                          aria-hidden="true"
+                          class="MuiOutlinedInput-notchedOutline emotion-44"
+                        >
+                          <legend
+                            class="emotion-45"
+                          >
+                            <span
+                              class="notranslate"
+                            >
+                              ​
+                            </span>
+                          </legend>
+                        </fieldset>
```

Comment:
Hmm, any idea why it added this? Usually this causes a failure when running against axeDevTools for the empty fieldset.

Reply:
Oh, did you happen to upgrade a yarn package locally, then run the update to snapshots before downgrading a package? I have had this happen to me before.

Reply:
Hmm yeah I did at one point early on upgrade Odyssey to 0.19 and then I rolled it back to 0.17. Could that be it?

Reply:
Ahh, yea, it like caches the updated version and the app references that version. I think I had to update the yarn lock from the topic branch and re-install or just delete the node_modules/odyssey folder and re-install and it should fix it.

Reply:
If that is the issue, bacon will also catch it 🙏